plugins/protocolbuffers/cpp/v26.0/Dockerfile

```diff
@@ -18,7 +18,10 @@ RUN bazel build '//:protoc_lib'
 COPY --link BUILD cpp.cc plugins/
 RUN bazel build '//plugins:protoc-gen-cpp.stripped'
 
-FROM gcr.io/distroless/cc-debian12:latest@sha256:e6ae66a5a343d7112167f9117c4e630cfffcd80db44e44302759ec13ddd2d22b
-COPY --from=build --link --chmod=0755 /build/bazel-bin/plugins/protoc-gen-cpp .
+FROM gcr.io/distroless/cc-debian12:latest@sha256:e6ae66a5a343d7112167f9117c4e630cfffcd80db44e44302759ec13ddd2d22b AS base
```

Comment:
Let's bump to the latest version from `.github/docker/Dockerfile.distroless-cc-debian` here (this is what our plugin fetcher uses to keep base images updated).